website/content/v1.10/advanced/selinux.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "SELinux"
+description: "SELinux security module support (experimental)."
+---
+
+Talos Linux 1.10 introduces initial SELinux support. Security-Enhanced Linux is a security module which
+helps mitigating attacks going past ordinary access control measures like file permission and container namespace isolation.
+
+## Configuration
+
+SELinux is enabled by default in Talos 1.10 images. The default mode is permissive, as currently
+some CNI and CSI solutions as well as extensions are incompatible with it. These missing parts
+are being worked on to make SELinux available for more installations.
+
+### Mode of operation
+
+> Please note that SELinux is still in an experimental state in Talos Linux. Extensions currently do not support enforcing mode, which is a known missing feature. Expect some CNI and CSI plugins to not work in enforcing mode. Please report the issues you encounter with different configurations to help cover various usage scenarios.
+
+> Enforcing mode should only be enabled on new installs as of version 1.10, since upgrade path enabling SELinux is still being worked on.
+
+SELinux can be put in enforcing mode (to actually prevent access when it is unauthorized by the policy) by adding `enforcing=1` to the kernel cmdline. This is most commonly done in UKI.
+
+## Development and internals
```

Comment:
I think we need to move this section into "Developing Talos" doc to make it less confusing, regular users should never touch it

up to section `SELinux Policy Structure` which should probably stay here to explain the policy, and probably we can expand on it a bit to explain our security boundaries

Reply:
Not sure if end users really want to dive into the policy structure, perhaps that also got to go to development docs